.bin/check_and_load_xml.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Can we get this file in `bin/` instead of `.bin/`? No need to hide the folder–it's critical to this project's functionality.